src/ruuvitag_ble/parser.py

```diff
@@ -1,16 +1,26 @@
 from __future__ import annotations
 
 import logging
+from typing import Type, Union
```

Comment:
Oh, these shouldn't be necessary since `from __future__ import annotations`. You can use as futuristic annotations as you like here, since they won't be parsed at runtime.